nall/chrono.hpp

```diff
@@ -3,14 +3,15 @@
 #include <nall/function.hpp>
 #include <nall/string.hpp>
 
+#include <chrono>
+
 namespace nall::chrono {
 
 //passage of time functions (from unknown epoch)
 
 inline auto nanosecond() -> u64 {
```

Comment:
MSVC doesn't implement this POSIX API. I initially had a _MSC_VER check here but I couldn't see any reason not to just use std::chrono everywhere. This also allowed me to stop linking against winpthreads in the existing Windows build configuration (which is where the MinGW implementation of this API resides) as it was our only dependency on that library.